session-3-backend-api/README.md

```diff
@@ -218,60 +214,52 @@ It formatted the request header and body, and send it to the server using the UR
 ## HTTP Status Code
 Ok, let's say you are uploading a message. How do you know that your message got successfully uploaded?
 
-Then, the __repsonse__ is going to help us!
+The __repsonse__ can tell us!
 
-A HTTP response is just like a request. 
-It has __header__ and __body__.
+A HTTP response is just like a request. It has a __header__ and __body__.
 
 
-In `header`, there is a field called `status` containing a number. That number indicates if your request was successful.
+In the `header`, there is a field called `status` containing a number. That number indicates if your request was successful.
```

Comment:
@GalenWong, in the future, let’s not abuse `monospace` font for concepts. Only use it for code.

Kevin, no need to change these things.